libjpeg-turbo/.build-test-rules.yml

```diff
@@ -0,0 +1,4 @@
+libjpeg-turbo/examples/hello_jpeg:
+  enable:
+    - if: (IDF_VERSION_MAJOR == 5 and IDF_VERSION_MINOR >= 4) and (IDF_TARGET in ["esp32", "esp32p4"])
```

Comment:
We don't have the P4 target in CI of idf-extra-components, so the "esp32p4" here doesn't do anything.